### ISSUE TITLE
Settings files pollute user's global namespace

### ISSUE THREAD

**Issue by breyed** [madskristensen/WebEssentials2012#5]
Web Essentials creates a folder for its settings in `%USERPROFILE%\Web Essentials`. This is equivalent to a user's global namespace.

Since Web Essentials' settings are fit for sharing among roaming profiles, they should live in `%APPDATA%\Web Essentials`.

**Comment by am11** [comment]
Or perhaps make it optional?

**Comment by breyed** [comment]
Options are expensive. More documentation, larger test matrix, steeper learning curve. Options make sense when there are compelling reasons for each option.

**Comment by madskristensen** [comment]
Yes, of course. Good idea. And you're right. No options for this is needed
